bin/cylc-check-software

```diff
@@ -211,14 +214,15 @@ def cmd_find_ver(
                        min_ver is not None else 'any')
     for cmd in cmd_base:
         try_next_cmd = True
-        if Popen(['which', cmd], stdin=open(os.devnull), stdout=PIPE,
-                 stderr=PIPE).wait():
+        if pcylc(['which', cmd], stdin=open(os.devnull),
+                 stdoutpipe=True, stderrpipe=True).wait():
             res = [NOTFOUND_MSG, False]
         else:
             try:
-                output = Popen([cmd, ver_opt], stdin=open(os.devnull),
-                               stdout=PIPE,
-                               stderr=PIPE).communicate()[outfile - 1].strip()
+                output = pcylc([cmd, ver_opt], stdoutpipe=True,
+                               stdin=open(os.devnull),
+                               stderrpipe=True).communicate()
```

Comment:
No biggie, but the `stdin` arg could go up a line - reduce line count by one :-)

Reply:
That causes the line length to be 87 and flags a pep8 error. In my opinion pep8 line length is too short.

Reply:
Sorry, you're right - I didn't notice the big indent last night.  Yeah the standard line length doesn't cope with multiple nesting/indent very well!